src/Files.App/UserControls/AddressToolbar.xaml

```diff
@@ -137,8 +138,45 @@
 				ScrollViewer.VerticalScrollBarVisibility="Hidden"
 				Text="{x:Bind ViewModel.PathText, Mode=OneWay}"
 				TextChanged="{x:Bind ViewModel.VisiblePath_TextChanged, Mode=OneWay}"
-				TextMemberPath="ItemPath"
-				Visibility="{x:Bind converters:MultiBooleanConverter.OrNotConvertToVisibility(ShowSearchBox, ViewModel.IsSearchBoxVisible), Mode=OneWay}" />
+				TextMemberPath="Text"
+				Visibility="{x:Bind converters:MultiBooleanConverter.OrNotConvertToVisibility(ShowSearchBox, ViewModel.IsSearchBoxVisible), Mode=OneWay}">
+				<AutoSuggestBox.ItemTemplate>
+					<DataTemplate x:DataType="items:NavigationBarSuggestionItem">
+						<StackPanel Margin="0,4">
+							<Grid>
+								<Grid.ColumnDefinitions>
+									<ColumnDefinition Width="*" />
+									<ColumnDefinition Width="Auto" />
+								</Grid.ColumnDefinitions>
+
+								<!--  Primary Display  -->
+								<!--  This is used to display paths or command descriptions.  -->
+								<TextBlock
+									Grid.Column="0"
+									Opacity="{x:Bind DisplayOpacity, Mode=OneWay}"
+									Text="{x:Bind PrimaryDisplay, Mode=OneWay}" />
+
+								<!--  Supplementary Display  -->
+								<!--  This is used to display command hotkeys.  -->
+								<TextBlock
```

Comment:
Can we use the standard theme brushes, instead of adding opacity to elements.

``TextFillColorSecondaryBrush`` or ``TextFillColorTertiaryBrush``

Reply:
Done.